support/kind/kind.go

```diff
@@ -153,7 +154,7 @@ func (k *Cluster) Destroy() error {
 
 func (k *Cluster) findOrInstallKind(e *gexe.Echo) error {
 	if e.Prog().Avail("kind") == "" {
-		log.Println(`kind not found, installing with GO111MODULE="on" go get sigs.k8s.io/kind@v0.11.0`)
+		log.V(4).Info(`kind not found, installing with GO111MODULE="on" go get sigs.k8s.io/kind@v0.11.0`)
```

Comment:
Use `Infof()` function to inject variable `KindVersion`

```go
log.V(4).Infof("kind not found, installing with GO111MODULE="on" go get sigs.k8s.io/kind@%s", kindVersion)
```

Reply:
Oh how silly of me. Totally missed this one. Thanks for the heads up.